lib/active_admin_versioning/active_admin/resource_controller.rb

```diff
@@ -18,6 +18,10 @@ def show
       protected
 
       def user_for_paper_trail
+        ActiveAdminVersioning.configuration.user_for_paper_trail ? self.send(ActiveAdminVersioning.configuration.user_for_paper_trail) : super
+      end
```

Comment:
I'm not expert in AA.

oh, since `current_admin_user` is configurable 
`config.current_user_method = :current_admin_user`
it is good Idea to do so, take a look at [AA base_controller](https://github.com/activeadmin/activeadmin/blob/master/lib/active_admin/base_controller.rb#L42), perhaps you could use `current_active_admin_user` and `current_active_admin_user?` methods.